packages/common/src/utils/alerting.utils.ts

```diff
@@ -36,11 +36,13 @@ export const addAlert = async (
 
   const {
     ACCOUNT_ID,
+    ALERT_MANAGER_ACCOUNT_ID,
     ALERT_MANAGER_REGION,
     ALERT_MANAGER_TOPIC,
   } = process.env;
 
-  const QUEUE_URL = `https://sqs.${ALERT_MANAGER_REGION}.amazonaws.com/${ACCOUNT_ID}/${ALERT_MANAGER_TOPIC}`;
+  const account_id = ALERT_MANAGER_ACCOUNT_ID || ACCOUNT_ID;
```

Comment:
suggestion(non-breaking): Add a comment or another more visible documentation informing that, if the `ALERT_MANAGER_ACCOUNT_ID` is not informed, the wallet service's own `ACCOUNT_ID` will be used as default to reach the Alert Manager.

Reply:
Added in https://github.com/HathorNetwork/hathor-wallet-service/pull/179/commits/967ddb2d7931d8f6202445eb8c02b0684f7e5b41.